src/core/src/CoreMain.py

```diff
@@ -27,7 +28,7 @@ def __init__(self, argv):
         composite_logger = bootstrapper.composite_logger
         stdout_file_mirror = bootstrapper.stdout_file_mirror
         telemetry_writer = bootstrapper.telemetry_writer
-        lifecycle_manager = status_handler = None
+        lifecycle_manager = status_handler = execution_config = None
```

Comment:
Important: consider that in auto-assessment, the configured auto-assessment invocation prior to new bits landing won't have the tmp folder in there. Review that that configuration is overwritten correctly as soon as new bits land. Or we'll see errors.

Reply:
Good point, I think I do see an issue here. 

Need to test this on a machine to better understand and validate

Reply:
This is not an issue for auto-assessment. You can ignore. The code already handles such changes at the first enable with the new code.